config/app.default.php

```diff
@@ -9,7 +9,7 @@
      * Development Mode:
      * true: Errors and warnings shown.
      */
-    'debug' => true,
+    'debug' => env('DEBUG', true),
```

Comment:
in the twisted PHP world at least in my book `var_dump((boolean) 'false'); // true`

e.g.
```php
<?php
var_dump((boolean) false); // false
var_dump((boolean) 'false'); // true
var_dump((boolean) '0'); // false
var_dump((boolean) 'NULL'); // true
var_dump((boolean) ''); // false